app/models/point_detail.rb

```diff
@@ -3,6 +3,8 @@ class PointDetail < ActiveRecord::Base
 
   belongs_to :principle
   belongs_to :point_request
+  has_many :point_detail_comments
+  before_destroy :destroy_associated_point_detail_comments
```

Comment:
Is the before_destroy necessary? I think it would be better if you define it in the relation.
`has_many :point_detail_comments, dependent: :destroy`

Or you can create a migration like this:
`add_foreign_key :point_details, :point_detail_comments, on_delete: :cascade`